test/e2e/app-dir/next-font/next-font.test.ts

```diff
@@ -47,10 +47,10 @@ describe('app dir - next/font', () => {
               // layout
               expect(JSON.parse($('#root-layout').text())).toEqual({
                 className: expect.stringMatching(
-                  process.env.TURBOPACK ? /^className_.*/ : /^__className_.*/
+                  process.env.TURBOPACK ? /.*_className$/ : /^__className_.*/
```

Comment:
I would prefer the formatting where `className` is in the front. That's much cleaner. Could we configure that for lightningcss?

Reply:
I think this was fixed on turbo side, so we can revert that on next.js side.